lighthouse-core/gather/connections/connection.js

```diff
@@ -109,6 +109,11 @@ class Connection {
   }
 
   _handleRawError(object, callback) {
+    // We proactively disable a few domains. Ignore any errors
+    if (object.error.message && object.error.message.includes('DOM agent hasn\'t been enabled')) {
```

Comment:
> will this suppress all legitimate errors when the domain isn't enabled?

if you mean will it suppress an error if you do something like `driver.sendCommand('DOM.getDocument')` when `DOM` hasn't been enabled, then no, the error will still come through. This error message only results from `DOM.disable` when already disabled